### PR TITLE
Bulk load experimental API

### PR DESCRIPTION
**Pull request by jakedt**

### CODE REVIEW

**Review by ecordell — CHANGES_REQUESTED** [review]

File: internal/services/v1/options/options.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package options
+
+import "time"
+
+//go:generate go run github.com/ecordell/optgen -output zz_generated.query_options.go . ExperimentalServerOptions
+
+type ExperimentalServerOptions struct {
+	StreamReadTimeount time.Duration `debugmap:"visible" default:"600s"`
```
[the remark this answers is not in the document]
Comment (jakedt) [reply internal/services/v1/options/options.go:8]: Which batch size? the client is completely in charge of batches...

[the remark this answers is not in the document]
Comment (ecordell) [reply internal/services/v1/options/options.go:8]: https://github.com/authzed/spicedb/pull/1281/files#diff-205cb7f3cb92dd867bf044800ab2c6e778af9af796d68a991229f24c12d19323R33

like this?

File: internal/services/v1/experimental.go [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+			newNamespaces[rel.Subject.Object.ObjectType] = struct{}{}
+		}
+		if rel.OptionalCaveat != nil {
+			if _, ok := existingCaveats[rel.OptionalCaveat.CaveatName]; !ok {
+				newCaveats[rel.OptionalCaveat.CaveatName] = struct{}{}
+			}
+		}
+	}
+
+	return lo.Keys(newNamespaces), lo.Keys(newCaveats)
```
[the remark this answers is not in the document]
Comment (jakedt) [reply internal/services/v1/experimental.go:136]: I'm using `lo` in other places. Should I remove those to drop the dependency? If not, is there any reason to prefer that one to the one in `lo`?

[the remark this answers is not in the document]
Comment (ecordell) [reply internal/services/v1/experimental.go:136]: just mentioning it, `lo` does way more than the package I linked, and you're using the more advanced things elswhere

File: internal/services/v1/options/options.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jakedt) [reply internal/services/v1/options/options.go:8]: Right... that would be a datastore flag. It's limited by a fixed size in spanner, mysql based on the configured sql value, and in postgres and cockroach there is no equivalent. I don't really know what's the best option here. The values I put in should just work™. Additionally, if we change our index structure in spanner, we may need to unilaterally reduce it...

**Review by ecordell — APPROVED** [review]
LGTM

**Pull request merged**